src/main/kotlin/no/nav/familie/oppdrag/grensesnittavstemming/GrensesnittavstemmingMapper.kt

```diff
@@ -107,7 +107,48 @@ class GrensesnittavstemmingMapper(
         }
     }
 
-    private fun opprettDetaljType(oppdrag: OppdragLager): DetaljType? =
+    private fun håndterAvstemmingstidspunkt(oppdrag: OppdragTilAvstemming) {
+        val fom = avstemmingstidspunkt.fom
+        val tom = avstemmingstidspunkt.tom
+        if (fom == null || fom > oppdrag.avstemmingTidspunkt) {
+            avstemmingstidspunkt.fom = oppdrag.avstemmingTidspunkt
```

Comment:
Dette skjønner jeg ikke helt. Vil fom og tom endre seg til noe annet enn inputdata?

Reply:
Metoden er for å plukke ut laveste/høyeste avstemmingstidspunktene fra listen over oppdrag

Reply:
Jeg skjønner ikke hvorfor du trenger det. Avstemmingstidspunkt ser ut til å være bruk i opprettPeriodeData, som brukes kun fra lagTotalMelding. Kunne man ikke likegodt brukt fom og tom fra input i totalMelding.  Hva skjer hos oppdrag hvis man tidligste er klokka 6 og seneste er klokka 23? Forventer ikke de et helt døgn?

Reply:
Jeg vet ikke helt hva som er forventet, jeg har bare beholdt den funksjonalitet som finnes fra før, men splittet opp henting og genrering av data.
Gjetter på at de ønsker første og siste tom av de meldinger vi sender over